[169054] Fix broken link (#76)
access-company/antikythera: doc_src/getting_started.md

- Congratulations! Now your first gear is up and running.
    - Take a look around and play around with it.
    - Check out documentations and see what you can do with antikythera and gears.

>          - Guide for instance administrators[](./instance_administrators/) (TBD)

Reviewer: Sorry, my suggestion seems to have included extra white space.
```suggestion
        - Guide for instance administrators[](./instance_administrators/) (TBD)
```

Author: I applied
120ddef37e199f7d181c71e5f59909767771204a

    - Take a look around and play around with it.
    - Check out documentations and see what you can do with antikythera and gears.

>         - [Guide for instance administrators](./instance_administrators/) (TBD)

Reviewer: Outside this PR, links to non-existent pages look unnatural.
Would you like to temporarily disable the link?
```suggestion
         - Guide for instance administrators[](./instance_administrators/) (TBD)
```

Author: Thank you for your suggestion.
I applied.
2bbe9751e3d6bda1c815d990c6fe7a555084ba21

>         - [Guide for gear developers (i.e. developers of web services)](./gear_developers/README.md)

Reviewer: I think that whenever possible, the link to the document should be a link to `hexdocs.pm`.
```suggestion
        - [Guide for gear developers (i.e. developers of web services)](https://hexdocs.pm/antikythera/gear_developers.html)
```

Reviewer 2: In my opinion, a link in a Markdown file should point to a file in the repository.
I expect that the link will be converted properly when generating hex document.

Reviewer: It may be useful when adding new documents, but this time it is unnecessary.
Links to the documentation in this repository are already standardized at `hexdocs.pm`. (e.g. 5022b90be93790bd77a90513cd8ff2d81c0b6abd)

Author: Thank you for your suggestion.
I applied.
047962fcbc7888702bb3a341b463b7f55c46d1b1
